### PR TITLE
Feature/cicd

### PR DESCRIPTION
**Pull request by marius-se**
- secrets are in place
- [GitHub CloudFront invalidation action](https://github.com/awact/cloudfront-action) is deprecated since Novemeber

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ Resources:
         # DefaultRootObject: index.html
         Aliases:
         - !Ref 'DomainName'
-        # - !Join [ '', [ 'www.', !Ref 'DomainName' ]]
+        - !Join [ '', [ 'www.', !Ref 'DomainName' ]]
```
Comment (0xTim) [inline stack.yaml]: We want a separate CF distribution for the www. domain backed by a different S3 bucket that redirects to the non-www version

Comment (marius-se) [reply stack.yaml]: I think we don't need a second CF distribution, I pointed both `www` and apex to the same CF distribution using Route 53 and it's working:
1. https://www.areweserveryet.org
2. https://areweserveryet.org

...or is there another reason for a separate CF+S3?

Comment (0xTim) [reply stack.yaml]: So the way we currently have it is that the apex site and www site are the same thing content wise but two different sites. That means for things like search results the ranking will be split over the two sites. By having one site that is the real site and then redirects on everything else it improves search results and there's only one site that everyone will end up on

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      with:
+        args: --acl public-read --follow-symlinks --delete
+      env:
+        AWS_S3_BUCKET: '${{ env.S3_BUCKET_NAME }}'
+        AWS_ACCESS_KEY_ID: ${{ secrets.DEPLOYER_AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.DEPLOYER_AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: '${{ env.S3_BUCKET_REGION }}'
+        SOURCE_DIR: 'Output'
+      if: steps.clouddeploy.outcome == 'success'
+    # - name: Invalidate CloudFront
```
Comment (0xTim) [inline .github/workflows/deploy.yaml]: We should find a substitute for this as we'll need to invalidate it

**Review by 0xTim — APPROVED** [review]
LGTM - last query for me is around DNS entries - is that manual or are we going to do it with the CF stack? (either is fine realistically)

**Comment by marius-se** [comment]
Ah I accidentally removed that in [a5edf1c](https://github.com/brokenhandsio/are-we-server-yet/pull/1/commits/a5edf1c3ac112e454377e1d8e149c34e99bcfd3d). It's back now!

**Review by 0xTim — APPROVED** [review]

**Pull request merged**